evergreen/evergreen.yml

```diff
@@ -600,6 +607,27 @@ tasks:
         - func: bootstrap-mongo-orchestration
         - func: run-tests
 
+    - name: test-netfw
+      commands:
+        - func: bootstrap-mongo-orchestration
+        - func: run-tests
+          vars:
+            FRAMEWORK: Net452
```

Comment:
Aren't TFM monikers all lower case?

    net452

Reply:
So in build.cake they are that way only because other Tasks are that way.

I think we can use standard TFM casing everywhere else.

We can either convert it to build.cake casing at the last minute or just rely on the fact that build.cake is not case sensitive when deciding which task to target.

Reply:
I went with keeping it lowercase and relying on the fact that build.cake is not case sensitive since I didn't want to muck around with changing casing inside bash for only cosmetic benefits :)